### ISSUE TITLE
Allow HTTP headers to define an environment (not only subdomains)

### ISSUE THREAD

**Issue by santiagofn** [ylecuyer/envify#12]
It'll be really cool if we can use certain HTTP header fields as parameters to identify an environment, and then not be restricted to use only subdomains.
(Real life example: at my work, we use custom HTTP request headers to set beta/rc enviroments).

**Comment by ylecuyer** [comment]
Good idea @santiagofn. This is doable I will work on it and come to you when it is done for some testing
